src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -46,6 +46,7 @@ object WorkspaceService {
   case class CloneWorkspace(sourceWorkspace: WorkspaceName, destWorkspace: WorkspaceName) extends WorkspaceServiceMessage
   case class GetACL(workspaceName: WorkspaceName) extends WorkspaceServiceMessage
   case class PutACL(workspaceName: WorkspaceName, acl: String) extends WorkspaceServiceMessage
+  case class SetupWorkspaceGroupACLs(workspaceName: WorkspaceName, bucketName: String) extends WorkspaceServiceMessage
```

Comment:
Why is there a message for this? I assume it's not going to have its own endpoint if it's something that wants to be done whenever a workspace is created. In that case, don't you want to call the function directly from clone/create workspace (as you are doing) - you only need a message if you're sending it across actors.